sycl/test/abi/sycl_symbols_linux.dump

```diff
@@ -3,7 +3,7 @@
 # DO NOT EDIT IT MANUALLY. Refer to sycl/doc/developer/ABIPolicyGuide.md for more info.
 ################################################################################
 
-# RUN: env LLVM_BIN_PATH=%llvm_build_bin_dir %python %sycl_tools_src_dir/abi_check.py --mode check_symbols --reference %s %sycl_libs_dir/libsycl.so
```

Comment:
why you removed it? I guess it was done like this for systems where python3 installed.

Reply:
That's what the script did.